tests/test_vision_explanation.py

```diff
@@ -193,3 +195,39 @@ def test_vision_explain_loadmodel():
     # delete files created during testing
     for elt in [savepath+"0"+".jpg", savepath2+"0"+".jpg"]:
         os.remove(elt)
+
+
+def test_vision_explain_evaluation():
+    """End to end testing for explanation evaluation."""
+    # pointing game run
+    imgpath = os.path.join('python', 'vision_explanation_methods', 'images',
+                           '128.jpg')
+    # load fastrcnn model
+    modelpath = os.path.join('python', 'vision_explanation_methods', 'models',
+                             'fastrcnn.pt')
+    # save tested result in res
+
+    # use helper function from above to fetch model
+    _ = download_assets(modelpath)
+
+    # run the main function for saliency map generation
+    device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+    model = _get_instance_segmentation_model(5)
+    model.load_state_dict(torch.load(modelpath, device))
+    model.to(device)
+    model.eval()
+    detection_model = PytorchDRiseWrapper(model=model,
+                                          number_of_classes=1000)
+
+    # find saliency scores for top 20% of salient pixels
+    # do this for the second object in an image
+    pg = PointingGame(detection_model)
+    s = pg.pointing_game(imgpath, 1, .8)
```

Comment:
Can the 1 and .8 be moved to some variables?  Is there a recommended default threshold that could be used?  It seems like that might be .8?